### PR TITLE
feat: use env var UNS_NETWORK to select network

### PR DESCRIPTION
**Pull request by peterjah**
#1429: Use UNS_NETWORK environment variable

### CODE REVIEW

File: src/baseCommand.ts [hunk 1]
```diff
@@ -48,6 +49,15 @@ export abstract class BaseCommand extends Command {
         /**
          * Configuration
          */
+
+        // This happen when providing network through env var
+        if (!UTILS.getNetworksList().includes(flags.network)) {
```
Comment (ftreguer) [inline src/baseCommand.ts:58]: This check is not provided by oclif? If UNS_NETWORK value is not included in options list, it should directly throw an exception  without calling Command.run()

**Review by ftreguer — CHANGES_REQUESTED** [review]

File: src/baseCommand.ts (hunk 1 above)
Comment (peterjah) [reply src/baseCommand.ts:58]: I know, but apparently the check is not done when the flag is set by environment variable. I guess its an oclif issue

Comment (peterjah) [reply src/baseCommand.ts]: PR has been pushed on Oclif project https://github.com/oclif/parser/pull/64

Comment (peterjah) [reply src/baseCommand.ts:58]: PR has been pushed on Oclif project oclif/parser#64

File: test/commands/status.test.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import { expect, test } from "@oclif/test";
 import { NETWORKS } from "../../src/config";
+process.env.DEV_MODE = "false";
```
Comment (ftreguer) [inline test/commands/status.test.ts]: You should not have to deal with it here :/ If DEV_MODE breaks your tests disable it

**Review by ftreguer — CHANGES_REQUESTED** [review]

File: src/baseCommand.ts [hunk 3]
```diff
@@ -49,6 +50,17 @@ export abstract class BaseCommand extends Command {
         /**
          * Configuration
          */
+
+        // This happen when providing network through env var
+        // see Oclif PR: https://github.com/oclif/parser/pull/64
+        // will be removed if PR is accepted
+        if (!UTILS.getNetworksList().includes(flags.network)) {
+            throw new Error(`Expected --network=${
```
Comment (ftreguer) [inline src/baseCommand.ts]: Not the same output: 
with native check: 
```
 ›   Error: Expected --network=lkmlk to be one of: devnet, local
 ›   See more help with --help
```

with your check:
```
Error: Expected --network=dalinet to be one of: [devnet|local]
            See more help with --help
    at StatusCommand.run (~/workspaces/uns-cli/src/baseCommand.ts:58:19)
    at StatusCommand._run (~/workspaces/uns-cli/node_modules/@oclif/command/lib/command.js:44:31)
```

**Review by ftreguer — CHANGES_REQUESTED** [review]

File: src/baseCommand.ts (hunk 3 above)
Comment (ftreguer) [reply src/baseCommand.ts]: See FlagInvalidOptionError : https://github.com/oclif/parser/blob/6577b9ecc9f34d7a78b46f17fa2176d90f5a9244/src/errors.ts#L84

**Review by ftreguer — APPROVED** [review]

**Pull request merged**